modules/alerting/main.tf

```diff
@@ -365,10 +365,10 @@ resource "google_logging_metric" "cloud-run-scaling-failure" {
 resource "google_monitoring_alert_policy" "cloud-run-scaling-failure" {
   # In the absence of data, incident will auto-close after an daily
   alert_strategy {
-    auto_close = "86400s"
+    auto_close = "3600s"
 
     notification_rate_limit {
-      period = "86400s" // re-alert daily if condition still valid.
+      period = "3600s" // re-alert daily if condition still valid.
```

Comment:
super nit: should this be `hourly` instead of `daily` now?

Reply:
thanks, fixed.